storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="persistent-storage-csi-smb-cifs"]
+= SMB CIFS CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: persistent-storage-csi-smb-cifs
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Server Message Block (SMB) protocol Common Internet File System (CIFS) dialect.
+
+:FeatureName: SMB CIFS CSI Driver Operator
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+After installing the SMB CIFS CSI Driver Operator, {product-title} installs the SMB CIFS CSI Operator and the SMB CIFS CSI driver by default in the `openshift-cluster-csi-drivers` namespace. This allows the SMB CIFS CSI Driver Operator to create CSI-provisioned PVs that mount to SMB CIFS assets.
```

Comment:
> After installing the SMB CIFS CSI Driver Operator, OpenShift Container Platform installs the SMB CIFS CSI Operator and the SMB CIFS CSI driver by default in the openshift-cluster-csi-drivers namespace. This allows the SMB CIFS CSI Driver Operator to create CSI-provisioned PVs that mount to SMB CIFS assets.

This sounds rather confusing to me. AFAIU, we have two concepts: "CSI Driver" and "CSI Driver Operator". It's hard to guess what's the difference between "SMB CIFS CSI Operator" and "SMB CIFS CSI Driver Operator". Also, it is CSI Driver (not CSI Driver Operator) who creates CSI-provisioned PVs that mount to SMB CIFS assets. I'd suggest the following text:

> After installing the SMB CIFS CSI Driver Operator, OpenShift Container Platform installs corresponding Pods for the operator and driver in the openshift-cluster-csi-drivers namespace by default. This allows the SMB CIFS CSI Driver to create CSI-provisioned PVs that mount to SMB CIFS assets.

Reply:
yes +1